draft-packages/stories/Menu.stories.tsx

```diff
@@ -190,8 +190,8 @@ const StatelessMenuExample = props => {
       </Paragraph>
       <StatelessMenu
         isMenuVisible={isMenuVisible}
-        toggleMenuDropdown={toggleMenu}
-        hideMenuDropdown={hideMenu}
```

Comment:
As you can see, only the StatelessMenu stories needed to be updated. The `Menu` stories are unchanged (were not affected by the breaking change)